website/pages/index.mdx

```diff
@@ -1,73 +1,18 @@
 ---
-title: Getting Started With GraphQL.js
-sidebarTitle: Getting Started
+title: Overview
+sidebarTitle: Overview
 ---
 
-{/* title can be removed in Nextra 4, since sidebar title will take from first h1 */}
+GraphQL.JS is the reference implementation to [The GraphQL Specification](https://spec.graphql.org/draft/), it's designed to be simple to use and easy to understand
+while closely following the Specification.
 
-# Getting Started With GraphQL.js
+You can build GraphQL servers, clients, and tools with this library, it's designed so you can choose which parts you use, for example, you can build your own parser
+and use the execution/validation from the library. There also a lot of useful utilities for schema-diffing, working with arguments, ...
```

Comment:
we should link to some page for more utility function definitions. Right now the `...` doesn't make a good ending
```suggestion
and use the execution/validation from the library. There also a lot of useful utilities for schema-diffing, working with arguments and [many more](LINK_SOMEWHERE).
```